lib/govuk_app_config/govuk_prometheus_exporter.rb

```diff
@@ -1,6 +1,6 @@
 module GovukPrometheusExporter
   def self.configure
-    unless Rails.env == "test"
+    unless Rails.env == "test" || (ENV["GOVUK_PROMETHEUS_EXPORTER"]).blank?
```

Comment:
Is there an explicit value that we expect "GOVUK_PROMETHEUS_EXPORTER" to be set to? I'm just wondering, is there any case where GOVUK_PROMETHEUS_EXPORTER is "false" or similar

Reply:
So thinking something like might have more predictable behaviour:
```suggestion
    unless Rails.env == "test" || (ENV["GOVUK_PROMETHEUS_EXPORTER"]) != "true"
```

This will make it explicit that we're expecting `GOVUK_PROMETHEUS_EXPORTER = true` to enable the prometheus exporter. If we check for `.blank?` then for example `GOVUK_PROMETHEUS_EXPORTER = false` would also enable the prometheus exporter.